python/lsst/ts/linearstage/csc.py

```diff
@@ -320,7 +341,10 @@ async def do_moveRelative(self, data):
         self.assert_notmoving("moveRelative")
         self.assert_target_in_range(data.distance, move_type="relative")
         await self.report_detailed_state(LinearStage.DetailedState.MOVINGSTATE)
-        await self.component.move_relative(data.distance)
+        try:
+            await self.component.move_relative(data.distance)
+        except Exception:
+            pass

```

Comment:
Again an exception is ignored. Please at least add an inline comment explaining why and what the expected behavior is.